### ISSUE TITLE
Deactivate streamer account for configurable time after manual disconnect

### ISSUE THREAD

**Issue by Vaalyn** [AzuraCast/AzuraCast#898]
**Is your feature request related to a problem? Please describe.**
When you disconnect a streamer via the station profile site the streamer gets disconnected but popular streaming software like for example VirtualDJ will automatically reconnect after the manual disconnect. This makes the disconnect streamer function not quite as reliable for kicking a dj off of a stream when he/she is doing something that needs to be stopped since he/she will just reconnect without any real delay or anything.

**Describe the solution you'd like**
I'd like to be able to configure an amount of time that the streamer account of a streamer that is manually disconnected will be deactivated for so that a manual disconnect forces a hard limit on when a dj can reconnect after he/she is thrown off the stream.

Depending on how granular the time should be configurable there needs to be some kind of cronjob that checks if the time that a streamer account should be deactivated has passed and activate the account if true. If it should only be configurable in minutes then an addition to the `syncShort` sync tasks would be enough. If it should be configurable in seconds than there needs to be a new kind of sync task that runs approximately every second.

The time should be configurable in the edit station profile screen.

To keep track of when a streamer account can be reactivated the easiest way would be to add a field to the `StationStreamer` entity called `reactivate_at` of type `int | nullable` where we set the timestamp of the moment where the account is allowed to be reactivated when the disconnect streamer function is executed. When the cronjob runs we get every `StationStreamer` that has a value for `reactivate_at` and check if the current timestamp is equal or higher than the value of `reactivate_at`, activate the account and set the `reactivate_at` back to null.

**Describe alternatives you've considered**
I can't really think of any good alternatives besides having a not changeable time limit for this but this feature would be kind of annoying if you can't configure it like you individually need it.

**Additional context**
Related to issue #670

**Comment by SlvrEagle23** [comment]
@Vaalyn Yep, this was pretty much exactly what I had in mind for dealing with the "disconnect streamer" issue not having any sort of meaningful permanence for streaming apps that automatically reconnect streamers.

I may go the approach of not necessarily having it be a static configuration but rather a dropdown on the "Disconnect Streamer" button that says "...for 5 minutes, ...for 1 hour, disable account" etc

**Comment by Vaalyn** [comment]
@SlvrEagle23 I'll send you a PR some time today with an implementation of the deactivation of the streamer after disconnecting and reactivation after the time passed. 👍

**Comment by NeddyPlays** [comment]
Ooh yes please make it so that it Disabled the streamers account when i click Disconnect Streamer.
